crates/server/src/catalog.rs

```diff
@@ -168,8 +168,8 @@ impl GraphCatalog {
             .ok_or_else(|| GraphCatalog::graph_not_found(graph_name))
     }
 
-    pub fn insert(&mut self, graph_name: String, graph: GraphType) {
-        self.graphs.insert(graph_name, graph);
+    pub fn insert(&mut self, graph_name: Cow<str>, graph: GraphType) {
```

Comment:
I think using `impl Into<Cow<str>>` would allow you to directly use either a `String` or a `&str` without having to call Cow::from everywhere.

Or, use `impl Into<String>`, get the same API and avoid String copy when the name already is a String, when we change the catalog to use String as key :)

Reply:
perfect, thx